util/geoJsonCenter.js

```diff
@@ -16,4 +20,4 @@ module.exports = function( geometry ){
   }
 
   return;
```

Comment:
returning `undefined` means the function is of type `void` which it isn't. if there is ever an expectation of the function returning something, it shouldn't return void at any point. `null` would be an appropriate substitution for an object when one cannot be created.